client/ngapp/views/directives/ooni-explorer-list-measurement.directive.html

```diff
@@ -0,0 +1,22 @@
+<div class="column probe-cc">
```

Comment:
@bnvk This directive is where the measurements are sketched out for the explorer.